spec/command_recorder_spec.rb

```diff
@@ -37,7 +37,7 @@ def before(env)
     end
 
     it "does not fail when reverting" do
-      migration = Class.new ::ActiveRecord::Migration do
+      migration = Class.new ::ActiveRecord::Migration[5.0] do
```

Comment:
Strange that `enabled_middleware` always returns false.  Are you sure?  (Actually it should return either a class or `nil`.  See `spec/support/enableable.rb`

The intent of the spec is to make sure that the middleware doesn't cause a `:down` migration to fail; I probably introduced this test to fix a bug.  In particular, it should exercise the code in `lib/schema_plus/indexes/active_record/migration/command_recorder.rb`